src/main/java/com/synopsys/integration/alert/component/scheduling/SchedulingDescriptorActionApi.java

```diff
@@ -49,8 +47,8 @@ public SchedulingDescriptorActionApi(final TaskManager taskManager) {
 
     @Override
     public FieldModel readConfig(final FieldModel fieldModel) {
-        final Optional<Long> accumulatorNextRun = taskManager.getDifferenceToNextRun(BlackDuckAccumulator.TASK_NAME, TimeUnit.SECONDS);
-        fieldModel.putField(SchedulingDescriptor.KEY_ACCUMULATOR_NEXT_RUN, new FieldValueModel(List.of(accumulatorNextRun.map(String::valueOf).orElse("")), true));
+        final String accumulatorNextRun = taskManager.getDifferenceToNextRun(OnDemandTask.TASK_NAME, TimeUnit.SECONDS).map(String::valueOf).orElse("");
```

Comment:
This won't work since the OnDemandTask does not do the accumulation from the provider(s).  We should have a countdown per provider accumulator.

Reply:
Ah, I see what I missed. I will change that to BD and Polaris. Do we want a timer for the onDemand task or no?

Reply:
We do not want a timer for the on demand task. It runs frequently enough that it should fine to not display it